lib/widgets/recent_dm_conversations.dart

```diff
@@ -109,16 +112,15 @@ class RecentDmConversationsItem extends StatelessWidget {
         //   new Intl.ListFormat('ja').format(['Chris', 'Greg', 'Alya'])
         //   // 'Chris、Greg、Alya'
         title = narrow.otherRecipientIds.map((id) => store.users[id]?.fullName ?? '(unknown user)').join(', ');
-        // TODO(#95) need dark-theme color
-        avatar = ColoredBox(color: const Color(0x33808080),
+        // TODO(#95) need proper dark-theme color (this is ad hoc)
+        avatar = ColoredBox(color: designVariables.groupDmConversationIconBg,
```

Comment:
Is this `TODO(#95)` subsumed by the one on this field in DesignVariables.dark?

Reply:
Oh yeah; thanks for the catch.